osgtest/tests/test_51_edgmkgridmap.py

```diff
@@ -12,6 +12,7 @@ def test_01_config_mkgridmap(self):
         core.config['edg.conf'] = '/usr/share/osg-test/edg-mkgridmap.conf'
 
         core.skip_ok_unless_installed('edg-mkgridmap', 'voms-admin-server')
+        self.skip_ok_unless(core.state['voms-admin.read-members'], 'Cannot read VO member list')
```

Comment:
My only thought about not changing it to `skip_bad` is that in the future, we may have tests that only rely on `voms-admin-server` and the `edg-mkgridmap` tests will show up as failures.